src/App.vue

```diff
@@ -180,10 +180,13 @@ export default {
         })
         .catch(() => this.toast.error(this.$t("error.create_plio")));
     },
-    onClose(event) {
+    onClose() {
```

Comment:
why remove the `event` argument? `event` later on will become undefined then right?